recipe/meta.yaml

```
@@ -20,20 +20,20 @@ requirements:
   host:
     - python
     - setuptools
-    - numpy >=1.11,!=1.14.0,!=1.15.3
-    - astropy >=2.0.3
+    - numpy>=1.11,!=1.14.0,!=1.15.3
```

```suggestion
    - numpy >=1.11,!=1.14.0,!=1.15.3
```
add spaces everywhere between package name and version constraint